dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/MetadataFieldRestRepository.java

```diff
@@ -247,10 +260,14 @@ protected MetadataFieldRest createAndReturn(Context context)
 
         if (isBlank(metadataFieldRest.getElement())) {
             throw new UnprocessableEntityException("metadata element (in request body) cannot be blank");
+        } else if (metadataFieldRest.getElement().contains(".")) {
+            throw new UnprocessableEntityException("metadata element (in request body) cannot contain dots");
         }
 
         if (isBlank(metadataFieldRest.getQualifier())) {
             metadataFieldRest.setQualifier(null);
+        } else if (metadataFieldRest.getQualifier().contains(".")) {
```

Comment:
As I suggested in the UI PR, I think we should also check for commas and spaces.  Those should not be allowed in the qualifier